cfy_manager/main.py

```diff
@@ -650,12 +651,14 @@ def _get_components_list():
     # Can't easily use list comprehension here because this is a list of lists
     ordered_components = []
     for service in ordered_services:
-        ordered_components.extend(SERVICE_COMPONENTS[service])
+        for component in SERVICE_COMPONENTS[service]:
+            if not components or component in components:
+                ordered_components.append(component)
     return ordered_components
 
 
-def _create_component_objects():
-    components_to_install = _get_components_list()
+def _create_component_objects(filter_components):
```

Comment:
This should maybe be include_components, as filter sounds like it might be excluding?

Reply:
yeah i think i'll just rename to that in all usages